packages/cli-kit/src/private/node/ui/components/Subdued.tsx

```diff
@@ -0,0 +1,16 @@
+import {Text} from 'ink'
+import chalk from 'chalk'
+import React from 'react'
+
+interface Props {
+  subdued: string
+}
+
+/**
+ * `Subdued` displays some text that represents a sub-process
```

Comment:
Is a `sub-process` semantically linked to subdued text? What if instead we said that `Subdued` displays some text with subdued colors?